lib/note/index.js

```diff
@@ -8,6 +8,7 @@ const { newCheckViewPermission, errorForbidden, responseCodiMD, errorNotFound, e
 const { updateHistory, historyDelete } = require('../history')
 const { actionPublish, actionSlide, actionInfo, actionDownload, actionPDF, actionGist, actionRevision, actionPandoc } = require('./noteActions')
 const realtime = require('../realtime/realtime')
+const serv = require('./service')
```

Comment:
rename service -> noteAlias, and consider using object destruction

Reply:
I would like to make the service is for all note controller.
And the service will contains the common logic in controller and easy for testing.